config.json

```diff
@@ -481,6 +481,22 @@
           "pids"
         ],
         "status": "wip"
+      },
+      {
+        "slug": "library-fees",
+        "name": "Library Fees",
+        "uuid": "b5629003-638c-47c0-ad76-a0769f285053",
+        "concepts": [
+          "dates-and-time"
+        ],
+        "prerequisites": [
+          "integers",
+          "floating-point-numbers",
+          "tuples",
+          "atoms",
+          "if"
```

Comment:
It's mentioned in the concept but not in the introduction about the erlang calendar module. From what I see in the exercise this isn't required in the exercise, but perhaps if there will be a locking component to concept progression then the erlang modules should be a pre-req to the concept?

Reply:
Yeah, the `calendar` module is not necessary at all for this exercise. I found that the Erlang `calendar` module is way less useful than I thought it would be, in that the Elixir stdlib already has a lot. My assumption was that it's ok for a `concept/about.md` to violate prerequisites a bit in the service of completeness. For example, in `processes/about.md`, `Agent` and `GenServer` are briefly mentioned. If you think that in this case it's too much, I think I would rather just remove the mention of `:calendar` and `to_erl` from this file and not have a prerequisite on `erlang-libraries`.

Reply:
You're more in touch with that than me, and if that is allowable in the other circumstances I think this is probably quite fine in this case.

Reply:
After a second thought, I decided that the Erlang date format is not so important in this context and just removed the mention from the `about.md` file.